### ISSUE TITLE
[BUG/ISSUE] PM25ss uses ORG_GROWTH rather than SSA_GROWTH

### ISSUE THREAD

**Issue by FeiYao-Edinburgh** [geoschem/geos-chem#558]
Hi GCST,

I thought I found a small bug in `aerosol_mod.F90`, in which `PM25ss` diagnostic is calculated using `ORG_GROWTH` but it should use `SSA_GROWTH`.
```
       ! PM2.5 sea salt
       IF ( State_Diag%Archive_PM25ss  ) THEN
          ! State_Diag%PM25ss(I,J,L) = ( SALA(I,J,L) * ORG_GROWTH  ) &
          State_Diag%PM25ss(I,J,L) = ( SALA(I,J,L) * SSA_GROWTH  ) &
                                   * ( 1013.25_fp  / PMID(I,J,L) ) &
                                   * ( T(I,J,L)    / 298.0_fp    ) &
                                   * 1.0e+9_fp
       ENDIF
```

**Comment by msulprizio** [comment]
Thanks for reporting this bug. We can fix it for the GEOS-Chem 13.0.0 release.

**Comment by msulprizio** [comment]
This is now resolved in 04bf01e2695059ee048f763272522e726d9dfb9f.
